test/testhelpers/nfs_persistentvolume.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: nfs
+  labels:
+    role: nfs-server
+spec:
+  capacity:
+    storage: 5Gi
+  accessModes:
+  - ReadWriteMany
+  storageClassName: nfs-server
+  nfs:
+    path: "/"
+    server: ""
+  persistentVolumeReclaimPolicy: Delete
+
```

Comment:
Let's move all of these manifests to test/e2e/manifests/nfs and deflate the `nfs_*` prefix to just the resource name.